tests/expected/function-contract/missing_contract_for_check.expected

```diff
@@ -0,0 +1,10 @@
+error: Failed to check contract: Function `harness` has no contract.
+  |
+7 | #[kani::proof_for_contract(no_contract)]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+note: Try adding a contract to this function or use the unsound `stub` attribute instead.
```

Comment:
I think you got your notes switched... I believe this note is for misusing stub_verified. The one for stub_verified seems to be the one you want here.